markdown/usage/troubleshooting.md

```diff
@@ -347,6 +348,26 @@ To fix this, you must clean the entirety of the run's `work/` directory e.g. wit
 
 `ctrl +z` is **not** a recommended way of killing a Nextflow job. Runs that take a long time to fail are often still running because other job submissions are still running. Nextflow will normally wait for those processes to complete before cleaning shutting down the run (to allow rerunning of a run with `-resume`). `ctrl + c` is much safer as it will tell Nextflow to stop earlier but cleanly.
 
+## Using a local version of iGenomes
+
+The iGenomes config file uses `params.igenomes_base` to make it possible to use a local copy of iGenomes. However, as custom config files are loaded after `nextflow.conf` and the `igenomes.config` has already been imported and parsed, setting `params.igenomes_base` in a custom config file has no effect and the pipeline will use the s3 locations by default.
+
+You can specify a local iGenomes path by either:
+
+- Setting the igenomes_base path in a configuration profile.
+
+    ```nextflow
+    params {
+      igenomes_base = '/<path>/<to>/<data>/igenomes'
+    }
+    ```
+
+- Specifying an `--igenomes_base` path in your executation command.
+
+    ```bash
+    nextflow run nf-core/<pipeline> --input <input> -c <config> -profile <profile> --igenoms_base <path>/<to>/<data>/igenomes
+    ```
+
```

Comment:
```suggestion

- Specifying the `igenomes_base` parameter in a `params` file provided with `-params-file in `yaml` or `json` format.

    ```bash
    nextflow run nf-core/<pipeline> -profile <profile> -params-file params.yml
    ```

    Where the `params.yml` file contains the pipeline params:

    ```bash
    input: '/<path>/<to>/<data>/input'
    igenomes_base: '/<path>/<to>/<data>/igenomes'
    ```
```

Reply:
Just to add also the option of providing an `igenomes_base` via a parameter file.

Reply:
Great point! I didn't think about this as an alternative option.